### PR TITLE
Basic implementation of anonymous sessions and quiz creation

### PR DESCRIPTION
**Pull request by Sierrawolf87**
The implementation of creating an anonymous user and the basic implementation of creating a quiz.
X-headers have been added, the database connection string has been updated.

### CODE REVIEW

File: BE.QuizByGPT/appsettings.Development.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost;Database=QuizByGPT-Dev;Trusted_Connection=True;Encrypt=false;TrustServerCertificate=True;Integrated Security=false;"
+    "DefaultConnection": "Server=79.137.204.168;Database=QuizByGPT-Dev;User={0};Password={1};Trusted_Connection=True;Encrypt=false;TrustServerCertificate=True;Integrated Security=false;"
```
Comment (ishakhorski-dev) [inline BE.QuizByGPT/appsettings.Development.json:9]: Is it makes sense to put server to secrets too? @Sierrawolf87

File: BE.QuizByGPT/appsettings.json [hunk 2]
```diff
@@ -5,5 +5,7 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=79.137.204.168;Database=QuizByGPT-Prod;User={0};Password={1};Trusted_Connection=True;Encrypt=false;TrustServerCertificate=True;Integrated Security=false;"
```
Comment (ishakhorski-dev) [inline BE.QuizByGPT/appsettings.json:9]: Same here

File: BE.QuizByGPT/appsettings.Development.json (hunk 1 above)
Comment (Sierrawolf87) [reply BE.QuizByGPT/appsettings.Development.json]: You're right. Changed it in the last commit.

File: BE.QuizByGPT/appsettings.json (hunk 2 above)
Comment (Sierrawolf87) [reply BE.QuizByGPT/appsettings.json]: Changed it in the last commit.

**Pull request merged**